From cf76f45f92a3820537c352aa63bd9bf93831cff8 Mon Sep 17 00:00:00 2001
From: Ma Wupeng <mawupeng1@huawei.com>
Date: Wed, 26 Apr 2023 16:02:14 +0800
Subject: [PATCH 042/263] [Huawei] mm: mem_reliable: Fix blank space issue in
 reliable_report_usage()

Offering: HULK
hulk inclusion
category: cleanup
bugzilla: https://gitee.com/openeuler/kernel/issues/I6WKXZ

--------------------------------

The blank space before kB is needed to align the previous memory report
style.

Signed-off-by: Ma Wupeng <mawupeng1@huawei.com>
---
 mm/mem_reliable.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/mem_reliable.c b/mm/mem_reliable.c
index 28baba72bb5e..d0db2909e93a 100644
--- a/mm/mem_reliable.c
+++ b/mm/mem_reliable.c
@@ -529,5 +529,5 @@ void reliable_report_usage(struct seq_file *m, struct mm_struct *mm)
 		return;
 
 	SEQ_PUT_DEC("Reliable:\t", atomic_long_read(&mm->reliable_nr_page));
-	seq_puts(m, "kB\n");
+	seq_puts(m, " kB\n");
 }
-- 
2.17.1

